pkg/modules/libreoffice/api/libreoffice.go

```diff
@@ -289,6 +289,14 @@ func (p *libreOfficeProcess) pdf(ctx context.Context, logger *zap.Logger, inputP
 		args = append(args, "--export", "ReduceImageResolution=false")
 	}
 
+	if options.ExportNotesInMargin {
+		args = append(
+			args,
+			"--export", "ExportNotes=true",
+			"--export", "ExportNotesInMargin=true",
```

Comment:
What is `ExportNotes` here 🤔?

Reply:
As it said in LibreOffice [documentation](https://help.libreoffice.org/latest/en-US/text/shared/guide/pdf_params.html): 
```ExportNotes - Specifies if notes are exported to PDF, boolean, default false```
So to be sure that we're exporting notes at all  to show them in margin I decided to add this overhead 😇
Am I wrong?

Reply:
I’m wondering is there are another kinds of notes than « margin » ones ^^ like would it make sense to have an `exportNotes` form fields?

Reply:
I've done some research and testing, there are at least two options for displaying notes/comments:
`ExportNotes` - show notes "inline", like a pinned note at the end of a modified sentence and
`ExportNotesInMargin` - show notes in the margins next to the modified line.

Sorry for the confusion :( I made the first changes hastily and didn't go deep into details.